node/cmd/ccq/utils.go

```diff
@@ -89,14 +89,14 @@ func validateRequest(logger *zap.Logger, env common.Environment, perms *Permissi
 	if err != nil {
```

Comment:
does `queryRequest.Unmarshal()` return error if `len(qr.QueryRequest)` is 0? If not would that unmarshal it into an empty object?

Reply:
it seems like `UnmarshalFromReader` does that. All good!

Reply:
Unmarshal from an empty buffer returns this error:
`failed to read message version: EOF`